Index: app/src/main/java/com/example/b07project/Presenter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.b07project;\r\n\r\nimport android.util.Log;\r\nimport android.view.View;\r\n\r\npublic class Presenter {\r\n\r\n    private Model model;\r\n    private LogInActivity view;\r\n\r\n    public Presenter(Model model, LogInActivity view) {\r\n        this.model = model;\r\n        this.view = view;\r\n    }\r\n\r\n    public void login(String email, String password) {\r\n\r\n        model.getUser(email, password, (User user) -> {\r\n            if (user == null) return;\r\n            if (user instanceof Customer)\r\n                view.redirectToCustomer(user.getId());\r\n            else\r\n                view.redirectToStoreOwner(user.getId());\r\n        });\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/b07project/Presenter.java b/app/src/main/java/com/example/b07project/Presenter.java
--- a/app/src/main/java/com/example/b07project/Presenter.java	
+++ b/app/src/main/java/com/example/b07project/Presenter.java	
@@ -18,9 +18,9 @@
         model.getUser(email, password, (User user) -> {
             if (user == null) return;
             if (user instanceof Customer)
-                view.redirectToCustomer(user.getId());
+                view.redirectToCustomer(user);
             else
-                view.redirectToStoreOwner(user.getId());
+                view.redirectToStoreOwner(user);
         });
     }
 }
