Index: app/src/main/java/com/example/b07project/Customer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.b07project;\n\npublic class Customer extends User{\n    public Customer(String email, String firstName, String lastName) {\n        super(email, firstName, lastName);\n    }\n\n    public Customer() {\n\n    }\n\n    @Override\n    public String toString() {\n        return firstName + lastName;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/b07project/Customer.java b/app/src/main/java/com/example/b07project/Customer.java
--- a/app/src/main/java/com/example/b07project/Customer.java	(revision 5a771bbdd3dadd8a902c1025e87403b25fd4891a)
+++ b/app/src/main/java/com/example/b07project/Customer.java	(date 1638116629001)
@@ -11,6 +11,6 @@
 
     @Override
     public String toString() {
-        return firstName + lastName;
+        return getFirstName() + getLastName();
     }
 }
